lib/ethpy/ethpy/hyperdrive/api/api.py

```diff
@@ -143,9 +143,8 @@ def __init__(
         self.hyperdrive_contract: Contract = web3.eth.contract(
             abi=abis["IHyperdrive"], address=web3.to_checksum_address(self.addresses.mock_hyperdrive)
         )
-        # TODO: in the future we want to switch to a single IERC4626Hyperdrive ABI
```

Comment:
This todo also encompassed switching away from `IHyperdrive`. I _think_ you can just do a slot-in replacement, everywhere we reference `IHyperdrive` can be switched to `IERC4626Hyperdrive`.